src/molecules/formfields/CreditCardField/credit_card_field.module.scss

```diff
@@ -0,0 +1,48 @@
+@import 'assets/stylesheets/global';
+
+$form_border: color('neutral-5');
+$padding: ru(.5);
+
+.credit-card {
+  // Place your component specific scss here.
+
+  &-header {
+    display: flex;
+    align-items: center;
+    padding-right: $padding;
+  }
+
+  &-lock {
+    width: ru(.75);
+    height: ru(.75);
+    margin-left: auto;
+  }
+
+  &-line1 {
+    position: relative;
+
+    border-bottom: 1px solid $form_border;
+  }
+
+  &-number {
+    width: 100%;
+  }
+
+  &-logo {
+    position: absolute;
+    top: 0;
+    right: $padding;
+    height: 100%;
+    pointer-events: none;
+  }
+
+  &-line2 {
+    display: flex;
+  }
+
+  &-exp, &-sec { width: 50%; }
+
+  &-exp {
```

Comment:
@patrickkim I know you love this set up with the `&-`. It's good, and it makes sense, but I honestly think it's unnecessarily verbose since we're using the css-modules. `.line2` won't conflict with any other classes since there's no global name space. 

The Upside to this pattern would be that, if we ever decided to throw out css-modules, we wouldn't need to convert any of our code. We should discuss this in standup.